src/index.html

```diff
@@ -7,8 +7,75 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"  rel="stylesheet"
+    >
```

Comment:
In this case this syntax will look better

```suggestion
    <link
      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"  
      rel="stylesheet"
    >
```